plop-templates/@react-aria/docs/useComponent.mdx.hbs

```diff
@@ -0,0 +1,69 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-aria/{{packageName}}';
+import {HeaderInfo, FunctionAPI, TypeContext, InterfaceType} from '@react-spectrum/docs';
+import packageData from '@react-aria/{{packageName}}/package.json';
+
+```jsx import
+import {use {{~componentName~}} } from '@react-aria/{{packageName}}';
+```
+
+---
+category: Category Name
```

Comment:
should we prompt for this in the plop commandline? We'd probably need to keep an updated list of the current in use Categories though...

Reply:
Hm, yeah I could look into adding this into the command line, not sure the best way to do the category list though (other than manually keeping track and updating it ourselves which feels gross)

Reply:
well, the docs can parse mdx files and has a plugin to parse this synatx, so we could update it pretty easily if we just wrote out that list during some step. determining when to run it is probably the hardest part. Would be great if we could hook it up to text completion in ide's, but don't know how hard that would be, anyhow, it's fine not to do it in this PR. Instead, maybe we can make it some easy to recognize pattern and throw an error in the docs build if we come across that pattern?

Reply:
I don't think people will know the category at the time they are using plop so more often than not, something radon is going to go in here. Maybe too much for new contributors?